src/data/markdown/translated-guides/en/02 Using k6/11 Environment variables.md

```diff
@@ -47,7 +45,15 @@ PS C:\k6> $env:MY_HOSTNAME="test.k6.io"; k6 run script.js
 
 </CodeGroup>
 
-or using an [`-e` / `--env` CLI flag](/using-k6/options#supply-environment-variables) (which will be the same for all platforms):
+> #### ⚠️ Default Settings
+>
+> Take note that by default, passing system environment variables does not work for `k6 archive`, `k6 cloud` and `k6 inspect`. In order for it to work, you'll need to explicitly specify [`--include-system-env-vars`](https://k6.io/docs/using-k6/options/#include-system-env-vars). You can also disable the default passing of system environment variables when running scripts by using `--include-system-env-vars=false`.
```

Comment:
I would add that disabling it is not recommended as it can lead to not intended env variables going to the cloud or getting into the resulting `archive.tar`. 
After all my `env` output is huge and on a server or a CI instance there will probably be at least a couple of secrets.

Reply:
I think it's better dropping the "disabling" line